cmd/ir2proxy/main.go

```diff
@@ -66,6 +66,7 @@ func run() int {
 			log.Warn(err)
 			return 1
 		}
+		outputYAML = bytes.ReplaceAll(outputYAML, []byte("  creationTimestamp: null\n"), []byte(""))
```

Comment:
please add a comment with a link to the issue to provide context to the people who come after us.